src/components/lib/slm/fprr.c

```diff
@@ -20,13 +22,45 @@ void
 slm_sched_fprr_execution(struct slm_thd *t, cycles_t cycles)
 { return; }
 
+// simply dump a core's task queue for debug usage
```

Comment:
I think that this is much more specific than this. shouldn't the comment be closer to:

```c
/*
 * Dump core 0's runqueue information periodically (and only from core 0). Will only do so after this function has been called a configurable number of times.
 */
```

Also, recall that comments should use `/* */`.